src/PartitionGraph.f90

```diff
@@ -111,6 +111,8 @@ MODULE PartitionGraph
     INTEGER(SIK) :: nGroups=0
     !> Number of partitioning methods
     INTEGER(SIK) :: nPart=0
+    !> Weighting factor used when normalizing computational weights
+    REAL(SRK) :: wtfactor=0.0_SRK
```

Comment:
Could this be initialized to 1.0, since that should be the default behavior?

Reply:
I thought about it, but changed it to `0.0` to reflect the uninitialized state.  When the graphtype is called, if there is no parameter present, it will get set to `1.0` by default.  That make sense and sound ok?

Reply:
Yeah, it makes sense. This sort of thing is done a lot in Futility & MPACT. I think my mind always just defaults to the c++ behavior where you construct objects, and these initial values are respected i.e shouldn't have uninitialized states.

Reply:
Gotcha.